browser-switch/src/main/java/com/braintreepayments/browserswitch/BrowserSwitchPersistentStore.java

```diff
@@ -0,0 +1,38 @@
+package com.braintreepayments.browserswitch;
+
+import android.content.Context;
+
+import org.json.JSONException;
+
+class BrowserSwitchPersistentStore {
+
+    private static final BrowserSwitchPersistentStore INSTANCE = new BrowserSwitchPersistentStore();
+
+    static BrowserSwitchPersistentStore getInstance() {
+        return INSTANCE;
+    }
+
+    private BrowserSwitchPersistentStore() {}
+
+    BrowserSwitchRequest getActiveRequest(Context context) {
+        BrowserSwitchRequest request = null;
+
+        String activeRequestJson = PersistentStore.get("browserSwitch", context);
```

Comment:
`browserSwitch` seems like a good candidate for a constant in this class. It could also be used to replace all the occurrences of this string within the tests.

Reply:
Yeah definitely will create a constant and update the class.